pontoon/insights/models.py

```diff
@@ -45,3 +45,15 @@ class Meta:
 
 class LocaleInsightsSnapshot(InsightsSnapshot):
     locale = models.ForeignKey("base.Locale", models.CASCADE)
+
+
+class ProjectInsightsSnapshot(AggregatedStats):
```

Comment:
I haven't made the new models inherit from `InsightsSnapshot` yet, because they are highly likely to change when we'll have the actual use case. That it will be a good opportunity to refactor all Insights models.